### PR TITLE
Implement multi-arch build for piraeus-client

### LINKED ISSUE #74: Build multi-arch images
Would like to use images that support multiple architectures for mixed clusters or at least additional images per architecture.

I'd like to contribute this but would need to understand the CI system for this repo.

More info here - https://docs.docker.com/engine/reference/commandline/manifest/

### PR DESCRIPTION
**Pull request by Jukie**
Closes #74

I didn't remove the Makefile with this PR in case that's used elsewhere. If it isn't we should be able to drop it.

### CODE REVIEW

File: .github/workflows/build-piraeus-client.yaml [hunk 1]
```diff
@@ -11,11 +11,36 @@ jobs:
...
+        id: prep
         run: |
+          DOCKERHUB_IMAGAE=docker.io/${{ secrets.DOCKERHUB_USERNAME }}/piraeus-client
+          QUAYIO_IMAGE=quay.io/${{ secrets.QUAYIO_USERNAME }}/piraeus-client
+        
           cd dockerfiles/piraeus-client
-          make update
+          . ./VERSION.env
+
+          TAGS="${DOCKERHUB_IMAGAE}:${LINSTOR_CLIENT_VERSION},${DOCKERHUB_IMAGAE}:${SHORT_VERSION}",${DOCKERHUB_IMAGAE}:latest,${QUAYIO_IMAGE}:${LINSTOR_CLIENT_VERSION},${QUAYIO_IMAGE}:${SHORT_VERSION}",${QUAYIO_IMAGE}:latest"
```
Comment (Jukie) [inline .github/workflows/build-piraeus-client.yaml:26]: Could probably clean this up

File: .github/workflows/build-piraeus-client.yaml [hunk 2]
```diff
@@ -11,11 +11,36 @@ jobs:
...
     steps:
-      - uses: actions/checkout@v2
-      - name: build images
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Prepare
+        id: prep
         run: |
+          DOCKERHUB_IMAGAE=docker.io/${{ secrets.DOCKERHUB_USERNAME }}/piraeus-client
```
Comment (rck) [inline .github/workflows/build-piraeus-client.yaml:20]: `IMAGAE`? Should not that be `IMAGE`

**Comment by rck** [comment]
I saw this got some force pushes. So I suggest you get it into a state you are satisfied with and then please squash the commits, this does not need to be a series. Then I'd suggest you add myself as reviewer and assign @WanzenBug , he did the CI/workflow work.

**Review by rck — CHANGES_REQUESTED** [review]
- see the previous comment for `IMAGAE` vs. `IMAGE`
- by squashing it like this you created a commit message combines the messages from the previous individual message with lots of S-o-b noise. Please rephrase

**Comment by Jukie** [comment]
Apologies for the delay here. Will finish my updates today.

**Comment by rck** [comment]
> Apologies for the delay here. Will finish my updates today.

That was 15 days ago. Do you still intend to work in this?

**Comment by toelke** [comment]
Hey, I would like to take this over (no idea what I'm getting in to). Is there a way for me to highjack this PR with my own branch or should I create a new one?

**Comment by rck** [comment]
> Hey, I would like to take this over (no idea what I'm getting in to). Is there a way for me to highjack this PR with my own branch or should I create a new one?

I'd say just create a new branch/MR and state in the description that this supersedes this ine

**Comment by toelke** [comment]
Wilco

**Comment by WanzenBug** [comment]
Closing this one, see #84

**Pull request closed without merging**